descopesdk/src/main/java/com/descope/internal/http/DescopeClient.kt

```diff
@@ -475,6 +475,13 @@ internal open class DescopeClient(internal val config: DescopeConfig) : HttpClie
         headers = authorization(refreshJwt),
     )
 
+    suspend fun logoutPrevious(refreshJwt: String) = post(
```

Comment:
Where is logoutAll?

Reply:
It's not implemented. I will take care of all cases in a different PR